dvtemplates/new_cdi_image.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+error_message() {
+    OS=$2
+    if [ $1 -eq 1 ]; then
+        echo -e "Unable to access data from the OS release URL"
+    elif [ $1 -eq 2 ]; then
+        echo -e "Invalid Image Version"
+    elif [ $1 -eq 3 ]; then
+        echo -e "Unable to connect to the OS Image repository"
+    elif [ $1 -eq 4 ]; then
+        echo -e "Container Image for the Latest $OS version is already present. Exiting"
+	exit 0
+    elif [ $1 -eq 5 ]; then
+        echo -e "Unable to push the image to the local Registry."
+    elif [ $1 -eq 6 ]; then
+        echo -e "Unable to push the image to the Quay Registry."
+    elif [ $1 -eq 7 ]; then
+        echo -e "End to End Tests fail."
+    elif [ $1 -eq 8 ]; then
+        echo -e "No OS Image is set. Exiting!"
+    fi
+    exit $1
+}
+
+trap 'error_message $?' EXIT
+
+#Insecure registry port
+port=5000
+
+# Check if TARGET_OS is set
+if [ -z "$TARGET_OS" ]; then
+    error_message 8
+fi
+
+FEDORA_OS="fedora"
+CENTOS="centos"
+
+# Set the Image registry and identify the latest version of the OS
+if [[ "${TARGET_OS}" == "${CENTOS}" ]]; then
+    cd ${PWD}/centos
+    OS_REPO="quay.io/shwetaap/centos-images"
```

Comment:
can you please change it to kubevirt repo?

Reply:
Done